control-plane/agents/src/bin/ha/cluster/switchover.rs

```diff
@@ -173,6 +176,19 @@ impl SwitchOverRequest {
         Ok(())
     }
 
+    /// Deletes Swtichover request from Etcd.
```

Comment:
Once request completes delete target . We update the stage as Successful. In that stage request is deleted from etcd. In case cluster agent restarts while processing some request. `fetch_incomplete_request` will pick them up and process them according to the last stage.

Reply:
I misunderstood your question. Do you want to keep requests that succeded in etcd?